github_connector/readme/USAGE.rst

```diff
@@ -0,0 +1,143 @@
+Initial upload from Github
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To fetch information from Github, you have to:
+
+#. go to 'Github' / 'Settings' / 'Sync Object'
+#. Select the object type you want to synchronize and its Gthub name
+
+   .. image:: /github_connector/static/description/sync_organization.png
```

Comment:
Please use relative path for proper visualization from all places (the same applies to all images):
```suggestion
   .. image:: ../static/description/sync_organization.png
```
README generator will replace these relative paths to proper absolute ones